assetsys.h

```diff
@@ -6355,6 +6425,57 @@ void test_assetsys( void ) {
     assetsys_destroy( assetsys );
 
     TESTFW_TEST_END();
+
+    TESTFW_TEST_BEGIN( "Test mounting data and loading file" );
+        {
+        // Zip file with a test.txt file containing "Hello, World!"
+        const char data[]  = {
```

Comment:
```suggestion
        const unsigned char data[]  = {
```
Looks like there's a build failing, I assume switching to unsigned char would fix it